doc/md/getting-started/04-ui.md

```diff
@@ -0,0 +1,111 @@
+---
+id: getting-started-management-ui
+title: Using the Management UI
+slug: /cli/getting-started/using-the-management-ui
+---
+
+### Using the Management UI
+
+Now that we have been introduced to the CLI’s capabilities, let’s see what we can do with the Management UI.
+
+###Inspecting
+We will start off by inspecting our database with the `-w` command. To do so, run:
+
+`atlas schema inspect -d "mysql://root:pass@tcp(localhost:3306)/example" -w`
+
+Our browser will open automatically, and we should see this output in the CLI:
+```
+Atlas UI available at: http://127.0.0.1:5800/projects/25769803777/schemas/1 
+Press Ctrl+C to stop
+```
+
+When the UI opens, we are taken to the schema page. This screen displays an overall look of our Atlas schema.
+
+![Atlas Schema Synced](https://atlasgo.io/uploads/images/docs/synced-atlas-schema.png)
+
+
+On the bottom-right, the schema is described using the Atlas DDL. On the top-right, the “Activity & History'' panel holds an audit history for all changes made to the schema. Each log is clickable, allowing us to view the diff between the schema and its previous state.
```

Comment:
When copying from word-processors sometimes these "ugly" quotes are used. In code we always use `"`
```suggestion
On the bottom-right, the schema is described using the Atlas DDL. On the top-right, the "Activity & History" panel holds an audit history for all changes made to the schema. Each log is clickable, allowing us to view the diff between the schema and its previous state.
```